src/nature/initialization_operators.cljc

```diff
@@ -32,6 +32,10 @@
           :fitness-score (fitness-function genetic-sequence))))
 
 (defn build-population
-  "Build `population-size` individuals by invoking `build-individual` on random, conforming genetic sequences."
-  [population-size alleles sequence-length fitness-function]
-  (repeatedly population-size #(build-individual (generate-sequence alleles sequence-length) fitness-function)))
+  "Build `population-size` individuals by invoking `build-individual` on random, conforming genetic sequences.
```

Comment:
This looks good, but I'd really appreciate a test with a trivial sequence-generator-function if you wouldn't mind